debian/rules

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
 
-PIP_INDEX_URL ?= https://pypi.python.org/simple
```

Comment:
wouldn't it be better to try to let it get this from the environment (I know this is a draft, so you maybe you were going to get to this anyway)

Reply:
This code already works that way.  The string after `?=` is the default value.  If `PIP_INDEX_URL` env var is defined, then that value will be used, otherwise `PIP_INDEX_URL` will be set to the default value:
```
$ cat Makefile
QQQ ?= 123

all:
	echo $(QQQ)
$ make
echo 123
123
$ QQQ=789 make
echo 789
789
```